taui/src/components/neighborhood-details.js

```diff
@@ -49,6 +51,47 @@ class NeighborhoodDetails extends PureComponent<Props> {
     }
   }
 
+  neighborhoodTrip (props) {
+    const { hasVehicle, listing, neighborhood, origin, userProfile, t } = props
+    // Look up the currently selected user profile destination from the origin
+    const originLabel = origin ? origin.label || '' : ''
+    const currentDestination = userProfile.destinations.find(d => originLabel.endsWith(d.location.label))
+    const bestJourney = listing ? (
+      listing.segments && listing.segments.length ? listing.segments[0] : null
+    ) : (
+      neighborhood.segments && neighborhood.segments.length ? neighborhood.segments[0] : null
+    )
+    const roundedTripTime = listing ? (Math.round(listing.time / ROUND_TRIP_MINUTES) * ROUND_TRIP_MINUTES) : (Math.round(neighborhood.time / ROUND_TRIP_MINUTES) * ROUND_TRIP_MINUTES)
```

Comment:
Moving the ternary further in would make this do the same calculation a little more succinctly:
```suggestion
    const roundedTripTime = Math.round((listing ? listing.time : neighborhood.time) / ROUND_TRIP_MINUTES) * ROUND_TRIP_MINUTES
```